model/src/main/java/oracle/kubernetes/operator/KubernetesConstants.java

```diff
@@ -6,7 +6,7 @@
 
 /** Kubernetes constants. */
 public interface KubernetesConstants {
-  String DEFAULT_IMAGE = "store/oracle/weblogic:12.2.1.3";
+  String DEFAULT_IMAGE = "store/oracle/weblogic:19.1.0.0";
```

Comment:
Same comment... Can't do this.

Reply:
https://github.com/oracle/weblogic-kubernetes-operator/pull/489/commits/4de0cf306c42f6d40fdbd24f1ea082930b83ec20